src/helm/proxy/models.py

```diff
@@ -448,18 +445,6 @@ def engine(self) -> str:
         name="huggingface/starcoder",
         tags=[CODE_MODEL_TAG],
     ),
-    # Local HuggingFace models
-    # Copy these models (in HuggingFace format) to ./huggingface_models
-    Model(
-        group="huggingface",
-        name="huggingface/llama-7b",
-        tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, LOCAL_HUGGINGFACE_MODEL_TAG],
-    ),
-    Model(
-        group="huggingface",
-        name="huggingface/alpaca-7b",
-        tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, LOCAL_HUGGINGFACE_MODEL_TAG],
-    ),
```

Comment:
I might have misunderstood this but where are these models defined now?

Reply:
I'm removing these two models. The first model can be replaced by `huggyllama/llama-7b`. There is no replacement for the Alpaca model, but the user can manually configure this model (as per the instructions in the PR description).